tests/labhub_test.py

```diff
@@ -53,6 +53,14 @@ def test_invite_cmd(self):
 
         self.assertEqual(self.labhub.TEAMS, self.teams)
 
+        testbot.assertCommand(
```

Comment:
as I suggested in DM, create a new test method, dont edit the existing one.

That way, the existing tests can not possibly break unless your code is wrong.

Reply:
They will break, since I'm introducing a new error on the functions present in the lanhub.py file. I'll have to either create a mock object which will ensure that the validate_nick function will return true or I can remove the error itself and just introduce the function and its test, but that won't solve the issue.

Reply:
I'm pushing the code, so that you can review better.